examples/connectivity/plot_adhd_tangent.py

```diff
@@ -0,0 +1,105 @@
+"""
+Comparing different connectivity measures
+=========================================
+
+This example compares different measures of functional connectivity between
+regions of interest.
+
+"""
+
+# Fetch dataset
+import nilearn.datasets
+atlas = nilearn.datasets.fetch_atlas_msdl()
+dataset = nilearn.datasets.fetch_adhd()
+
+# Extract and preprocess regions time series
+import nilearn.input_data
+masker = nilearn.input_data.NiftiMapsMasker(
+    atlas.maps, resampling_target="maps", detrend=True,
+    low_pass=None, high_pass=None, t_r=2.5, standardize=False,
+    memory='nilearn_cache', memory_level=1)
+subjects = []
+for func_file in dataset.func:
+    time_series = masker.fit_transform(func_file)
+    subjects.append(time_series)
+
+# Estimate connectivity matrices
+import nilearn.connectivity
+subjects_connectivity = {}
+measures = ['tangent', 'partial correlation', 'precision', 'correlation',
+            'covariance']
+mean_connectivity = {}
+from sklearn.covariance import EmpiricalCovariance
+for measure in measures:
+    estimator = {'measure': measure, 'cov_estimator': EmpiricalCovariance()}
+    cov_embedding = nilearn.connectivity.CovEmbedding(**estimator)
+    subjects_connectivity[measure] = nilearn.connectivity.vec_to_sym(
+        cov_embedding.fit_transform(subjects))
+    # Compute the mean connectivity across all subjects
+    if measure == 'tangent':
+        mean_connectivity[measure] = cov_embedding.tangent_mean_
+    else:
+        mean_connectivity[measure] = \
+            subjects_connectivity[measure].mean(axis=0)
+
+# Plot the mean connectivity
+import numpy as np
+import nilearn.plotting
+labels = np.recfromcsv(atlas.labels)
```

Comment:
In [2]: %run plot_adhd_tangent.py
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
/home/bertrand/mygit/nilearn/examples/connectivity/plot_adhd_tangent.py in <module>()
     46 import numpy as np
     47 import nilearn.plotting
---> 48 labels = np.recfromcsv(atlas.labels)
     49 region_coords = np.vstack((labels['x'], labels['y'], labels['z'])).T
     50 for measure in ['tangent', 'correlation', 'partial correlation']:

/usr/local/lib/python2.7/dist-packages/numpy/lib/npyio.pyc in recfromcsv(fname, **kwargs)
   1904     kwargs.setdefault("delimiter", ",")
   1905     kwargs.setdefault("dtype", None)
-> 1906     output = genfromtxt(fname, **kwargs)
   1907 
   1908     usemask = kwargs.get("usemask", False)

/usr/local/lib/python2.7/dist-packages/numpy/lib/npyio.pyc in genfromtxt(fname, dtype, comments, delimiter, skiprows, skip_header, skip_footer, converters, missing, missing_values, filling_values, usecols, names, excludelist, deletechars, replace_space, autostrip, case_sensitive, defaultfmt, unpack, usemask, loose, invalid_raise)
   1686             # Raise an exception ?
   1687             if invalid_raise:
-> 1688                 raise ValueError(errmsg)
   1689             # Issue a warning ?
   1690             else:

ValueError: Some errors were detected !
    Line #2 (got 5 columns instead of 4)
    Line #3 (got 5 columns instead of 4)
    Line #4 (got 5 columns instead of 4)
    Line #5 (got 5 columns instead of 4)
    Line #6 (got 5 columns instead of 4)
    Line #7 (got 5 columns instead of 4)
    Line #8 (got 5 columns instead of 4)
    Line #9 (got 5 columns instead of 4)
    Line #10 (got 5 columns instead of 4)
    Line #11 (got 5 columns instead of 4)
    Line #12 (got 5 columns instead of 4)
    Line #13 (got 5 columns instead of 4)
    Line #14 (got 5 columns instead of 4)
    Line #15 (got 5 columns instead of 4)
    Line #16 (got 5 columns instead of 4)
    Line #17 (got 5 columns instead of 4)
    Line #18 (got 5 columns instead of 4)
    Line #19 (got 5 columns instead of 4)
    Line #20 (got 5 columns instead of 4)
    Line #21 (got 5 columns instead of 4)
    Line #22 (got 5 columns instead of 4)
    Line #23 (got 5 columns instead of 4)
    Line #24 (got 5 columns instead of 4)
    Line #25 (got 5 columns instead of 4)
    Line #26 (got 5 columns instead of 4)
    Line #27 (got 5 columns instead of 4)
    Line #28 (got 5 columns instead of 4)
    Line #29 (got 5 columns instead of 4)
    Line #30 (got 5 columns instead of 4)
    Line #31 (got 5 columns instead of 4)
    Line #32 (got 5 columns instead of 4)
    Line #33 (got 5 columns instead of 4)
    Line #34 (got 5 columns instead of 4)
    Line #35 (got 5 columns instead of 4)
    Line #36 (got 5 columns instead of 4)
    Line #37 (got 5 columns instead of 4)
    Line #38 (got 5 columns instead of 4)